js/test/durfmt/testdurfmt2.js

```diff
@@ -9443,6 +9443,71 @@ module.exports.testdurfmt2 = {
         test.equal(clockformatted_14[2], '14 နာရီ 14 မိနစ် 14 s');
         test.equal(clockformatted_14[3], '14 နာရီ 14 မိနစ် 14 s');
 
+        test.done();
+    },
+    testDurFmt_mt_MT: function(test) {
+        test.expect(32);
+        // 1 5 14 100
+        var textfmt;
+        var data = setVariable();
+        var length = data["fullLength"];
+
+        var textformatted_1 = [], textformatted_5 = [], textformatted_14 = [], textformatted_100 = [];
+        var clockformatted_1 = [], clockformatted_5 = [], clockformatted_14 = [], clockformatted_100 = [];
+
+        for (var i=0; i<4; i++) {
+            textfmt = new DurationFmt({locale: "mt-MT", style:"text", length:length[i], useNative: false});
+            textformatted_1.push(textfmt.format({year: 1, month: 1, week: 1, day: 1}).toString());
+            textformatted_5.push(textfmt.format({year: 5, month: 5, week: 5, day: 5}).toString());
+            textformatted_14.push(textfmt.format({year: 14,month: 14,week: 14,day: 14}).toString());
+            textformatted_100.push(textfmt.format({year: 100, month: 100, week: 100, day: 100}).toString());
+
+            clockformatted_1.push(textfmt.format({hour: 1, minute: 1, second: 1}).toString());
+            clockformatted_5.push(textfmt.format({hour: 5, minute: 5, second: 5}).toString());
+            clockformatted_14.push(textfmt.format({hour: 14, minute: 14, second: 14}).toString());
+            clockformatted_100.push(textfmt.format({hour: 100, minute: 100, second: 100}).toString());
+        }
+
+        test.equal(textformatted_1[0], '1 y, 1 m, 1 w, u 1 d');
```

Comment:
Translations are available for Maltese on Google translate as well:

1 second = 1 sekonda
2 seconds = 2 sekondi
1 minute = 1 minuta
2 minutes = 2 minuti
1 hour = 1 siegħa
2 hours = 2 sigħat
1 day = 1 jum
2 days = Jumejn
3 days = 3 ijiem
1 week = Ġimgħa
2 weeks = Ġimagħtejn
3 weeks = 3 ġimgħat
1 month = Xahar
2 months = Xahrejn
3 months = 3 xhur
1 year = Sena
2 years = Sentejn
3 years = 3 snin

Reply:
As you wrote above, When I check the google translate, it looks like they have `two` category ;( 
2 days = Jumejn
3-10 days = {n} ijiem
14 days = {n}-il jum
100 days = {n} jum

Probably I need to report to CLDR.